test/etc/profile.definitions/site/EasyBuild.yml

```diff
@@ -0,0 +1,5 @@
+## help EasyBuild
+EASYBUILD:
+  BUILDPATH:     /dev/shm/$USER
+  MODULES_TOOL:  Lmod
+  MODULE_SYNTAX: Tcl
```

Comment:
Why `Tcl` is you use `Lmod`?

Reply:
ah, yes. it's there for compatibility purposes: we create modulefiles which may get consumed by non-Lmod platforms. Not much we can do about it at present, that's a sysadmin's life.